client/app/components/Form/index.jsx

```diff
@@ -1,5 +1,6 @@
 // @flow
 import React from 'react';
+import type { Node } from 'react';
```

Comment:
Similar comment as above about `useState`!

Reply:
Unfortunately, I cannot import type like this. See below:

![image](https://user-images.githubusercontent.com/25817917/76082271-3fa44d00-5fab-11ea-8c65-02ac84df52ba.png)